Sources/main.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+import Kitura
+
+import KituraStencil
+
+import LoggerAPI
+import HeliumLogger
+
+// Create a new router
```

Comment:
All of the comments in the file describe exactly what the next line does (except maybe the logger comments). What's the reason for these comments? 

Comments that explain what code does means we have multiple sources of truth (the comment and the line itself), which gives us more of a maintenance burden.

Is it to show an example for those who aren't familiar with the framework/Swift? Would the documentation for these frameworks be a better source of that?